common/build/eslint-config-fluid/CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # @fluidframework/eslint-config-fluid Changelog
 
+## [5.4.0](https://github.com/microsoft/FluidFramework/releases/tag/eslint-config-fluid_v5.4.0)
+
+### Disabled rules
+
+The following rules have been disabled in all configs because they conflict with formatter settings:
+
+-   [@typescript-eslint/brace-style](https://typescript-eslint.io/rules/brace-style)
+-   [unicorn/number-literal-case](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v48.0.1/docs/rules/number-literal-case.md)
+
+### @typescript-eslint/explicit-function-return-type changes
```

Comment:
Thanks for adding these! And sorry, I'll have to remember to do this when I make changes.

Reply:
I need to remember it for code review too :)